services/datamanager/builtin/builtin.go

```diff
@@ -532,6 +556,15 @@ func (svc *builtIn) Reconfigure(
 			svc.closeSyncer()
 		}
 	}
+
+	if !svc.captureDisabled {
+		fileDeletionCtx, cancelFunc := context.WithCancel(context.Background())
```

Comment:
Do we want the passed in context or the background context?

Reply:
The reason I use the background context is the context that is passed into reconfigure is closed shortly after. In my testing, the context was being cancelled from outside reconfigure and would stop the polling thread so I used the background to ensure I would cancel it only when intended outside of reconfigure.